app/services/file_manager/file_upload/upload_client.py

```diff
@@ -471,5 +473,32 @@ def check_status(self, file_objects: list[FileObject]) -> list[FileObject]:
 
         return unfinished_files
 
+    def upload_status_check(self, file_objects: list[FileObject]) -> None:
```

Comment:
I think we can directly naming it as this and remove the first line of the function
```suggestion
    def upload_status_check(self, unfinished_files: list[FileObject]) -> None:
```